### DISCUSSION TITLE
Different drives saves

### DISCUSSION THREAD

**Discussion by RACE4EVER** [mtkennerly/ludusavi#202] [Q&A]
Hello again
I'm trying to sync Starbound saves, but on my PC it's installed on `E` drive and on my laptop on `D` drive. And obviously it saves in backup directory into 2 different folders corresponding drive letter

Please hep with merging it

UPD: I've used redirection again and it appears it can link to non-existing drives, so it successfully saves files where i need them

Then another question appeared. Is it possible to make backups drive independent? Like without drive-C thing?

**Comment by mtkennerly** [comment]
Hi! Glad you were able to solve this with redirects :+1:

> Then another question appeared. Is it possible to make backups drive independent? Like without drive-C thing?

One way is to redirect the entire drive to a dummy drive, with different redirects for backup and restore:

> <img width="576" alt="image" src="https://github.com/mtkennerly/ludusavi/assets/4934192/bc12c46e-eb25-414c-a872-36eae1043edc">

There are some problems with this, though:

* You can only restore to one drive at a time. In the example screenshot, you can back up from C and D, but only restore to C. Since everything gets backed up as Z, Ludusavi won't know the original drive letter.
* If the same game is installed in the same place on two different drives, they'll be marked as duplicates.
